internal/services/ticker/ticker.go

```diff
@@ -222,6 +222,19 @@ func (t *TickerImpl) Start() (func() error, error) {
 		return nil, fmt.Errorf("could not schedule tenant alert polling: %w", err)
 	}
 
+	// poll for expiring tokens every 1 minutes
+	_, err = t.s.NewJob(
+		gocron.DurationJob(time.Minute*1),
```

Comment:
Once a minute seems very high for this process?

Reply:
increased to 15m